src/api/math/float/fmae.rs

```diff
@@ -0,0 +1,60 @@
+//! Implements vertical (lane-wise) floating-point `fmae`.
+
+macro_rules! impl_math_float_fmae {
+    ([$elem_ty:ident; $elem_count:expr]: $id:ident | $test_tt:tt) => {
+        impl $id {
+            /// Fused multiply add estimate: ~= `self * y + z`
+            ///
+            /// On some architectures, it is possible to combine a multiply
```

Comment:
I would just state here that:

> While fused-multiply add ([`fma`]) has infinite precision, [`fmae`] has at worst the same precision of a multiply followed by an add. This might be more efficient on architectures that do not have an `fma` instruction.

And leave it at that. I also think that we can remove the note about the fused-multiply subtract. While it wasn't obvious to me, its pretty obvious in hindsight that any of the values can be negative.